src/views/Home/index.tsx

```diff
@@ -1,25 +1,112 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+
 import { withAuth } from '@okta/okta-react';
 
 import useAuth from 'hooks/useAuth';
 import Header from 'components/Header';
+import Button from 'components/shared/Button';
+import ActionBanner from 'components/shared/ActionBanner';
+import { AppState } from 'reducers/rootReducer';
+import { fetchSystemIntakes } from 'types/routines';
+import { SystemIntakeForm } from 'types/systemIntake';
+import './index.scss';
 
-type HomeProps = {
+type HomeProps = RouteComponentProps & {
   auth: any;
 };
 
-const Home = ({ auth }: HomeProps) => {
+const Home = ({ auth, history }: HomeProps) => {
   const [isAuthenticated] = useAuth(auth);
+  const dispatch = useDispatch();
+  const systemIntakes = useSelector(
+    (state: AppState) => state.systemIntakes.systemIntakes
+  );
+  useEffect(() => {
+    if (isAuthenticated) {
+      dispatch(fetchSystemIntakes());
+    }
+  }, [dispatch, isAuthenticated]);
+
+  const getSystemIntakeBanners = () => {
+    return systemIntakes.map((intake: SystemIntakeForm) => {
+      switch (intake.status) {
+        case 'DRAFT':
+          // TODO: When content sweep gets merged, this needs to be requestName
+          return (
+            <ActionBanner
+              key={intake.id}
+              title={
+                intake.projectName
+                  ? `${intake.projectName}: Intake Request`
+                  : 'Intake Request'
+              }
+              helpfulText="Your Intake Request is incomplete, please submit it when you are ready so that we can move you to the next phase"
+              onClick={() => {
+                history.push(`/system/${intake.id}`);
+              }}
+              label="Go to Intake Request"
+            />
+          );
+        default:
```

Comment:
Are adding the other cases here or saving them for a future story?

Reply:
I think this is the only case for System Intake banners. 

I think for `case 'SUBMITTED'`, that triggers the `Start my Business Case` banner, so that would be added here in the business case banners. I chose to do it this way because it would be difficult to verify on IMPL when the status is `SUBMITTED` because we haven't implemented submitting the form yet. We'd need to manually update values in the DB.